### PR TITLE
Currency: Added build-script and build_info.json for type-parser

### PR DESCRIPTION
**Pull request by pooja0805**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by pooja0805** [comment]
Verification Log: [TypeParserScriptLog.txt](https://github.com/ppc64le/build-scripts/files/11177355/TypeParserScriptLog.txt)

File: t/type-parser/build_info.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "package_name": "type-parser",
+    "github_url": "https://github.com/drapostolos/type-parser.git",
+    "version": "type-parser-0.8.1",
+    "package_dir": "t/type-parser/",
+    "build_script": "type-parser_ubi_8.5.sh",
+	"validate_build_script": true,
```
Comment (nikhil-kalbande) [inline t/type-parser/build_info.json:7]: Please correct indentation in json and script file file

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]
Please correct indentation in all files

File: t/type-parser/build_info.json (hunk 1 above)
Comment (pooja0805) [reply t/type-parser/build_info.json]: Updated.

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**